packages/client/css/generic/input.css

```diff
@@ -0,0 +1,66 @@
+.Input {
+  margin-bottom: 16px;
+
+  & label {
+    display: block;
+    color: var(--gray-300);
+
+    font-weight: bold;
+    font-size: 16px;
+
+    text-transform: uppercase;
+    margin-bottom: 4px;
+    margin-left: 13px;
+
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
```

Comment:
Indentation doesn't quite line up. This problem probably also existed in the old CSS, but it should be fixed while we're in the process of reworking things.